docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds to `push` events originating from Docker Hub repositories.
+
+In response to a `push` event, the receiver "refreshes" `Warehouse` resources subscribed to the corresponding Docker Hub repository.
+
+:::info
+"Refreshing" a `Warehouse` means enqueuing it for immediate reconciliation by the Kargo controller, which will attempt to discover new artifacts from all subscribed repositories.
+:::
+
+:::note
+Unlike GitHub, Docker Hub does **not** sign webhook payloads. Kargo uses a static token for basic validation.
```

Comment:
Referring to GitHub here does not make much sense IMHO, as the user is interested in configuring DockerHub (and may not be aware of GitHub _at all_).